test/e2e/flask/btc/btc-send.spec.ts

```diff
@@ -0,0 +1,165 @@
+import { strict as assert } from 'assert';
+import { Suite } from 'mocha';
+import { Driver } from '../../webdriver/driver';
+import { DEFAULT_BTC_ACCOUNT, DEFAULT_BTC_BALANCE } from '../../constants';
+import {
+  getTransactionRequest,
+  SendFlowPlaceHolders,
+  withBtcAccountSnap,
+} from './common-btc';
+
+export async function startSendFlow(driver: Driver, recipient?: string) {
+  // Wait a bit so the MultichainRatesController is able to fetch BTC -> USD rates.
+  await driver.delay(1000);
+
+  // Start the send flow.
+  const sendButton = await driver.waitForSelector({
+    text: 'Send',
+    tag: 'button',
+    css: '[data-testid="coin-overview-send"]',
+  });
+
+  // Firefox test is flaky without this delay. The send flow doesn't start properly.
+  if (driver.browser === 'firefox') {
+    await driver.delay(1000);
+  }
+
```

Comment:
Nit: Just keep the `sendButton.click` on the same "block" than its selector + Adds a `FIXME` cause this will need to be removed some day:

```suggestion
  // FIXME: Firefox test is flaky without this delay. The send flow doesn't start properly.
  if (driver.browser === 'firefox') {
    await driver.delay(1000);
  }
```